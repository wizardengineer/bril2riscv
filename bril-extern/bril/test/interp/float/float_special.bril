@main {
  v0: float = const 0.0;
  v1: float = const -0.0;
  res: bool = feq v0 v1;
  print v0;
  print v1;
  print res;
  v2: float = const 1e6;
  v3: float = const 1E-6;
  v4: float = const -1E+6;
  v5: float = const +.11e12;
  print v2;
  print v3;
  print v4;
  print v5;
  v6: float = const 1e21;
  v7: float = const 1e-21;
  print v6;
  print v7;
  v8: float = const 1e5;
  print v8;
}
