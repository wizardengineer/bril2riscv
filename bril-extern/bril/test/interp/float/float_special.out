0.00000000000000000
-0.00000000000000000
true
1000000.00000000000000000
0.00000100000000000
-1000000.00000000000000000
1.10000000000000000e+11
1.00000000000000000e+21
9.99999999999999908e-22
100000.00000000000000000
